BitmovinAnalyticsCollector/Classes/AVPlayer/Classes/AVPlayerAdapter.swift

```diff
@@ -164,7 +158,10 @@ class AVPlayerAdapter: NSObject, PlayerAdapter {
                 NSLog("Current Item Changed: %@", currentItem.debugDescription)
                 startMonitoringPlayerItem(playerItem: currentItem)
             }
-
+        } else if keyPath == #keyPath(player.status) {
+            if(player.status == .failed) {
```

Comment:
Move this check inside of the `else if`, no need for it to be here and indent the code even further to the right.

Reply:
My thinking here was: if there are multiple `} else if keyPath == xxx {` down the line, all the other `else if` cases would still be checked if `keyPath = status` and `player.status` is not `.failed`. Although it's already clear that `keyPath` can't be anything else than `status` in this case.

Reply:
I still don't get why you did it this way 😅